core/src/test/java/org/wildfly/extras/sunstone/tests/AbstractCloudProviderTest.java

```diff
@@ -155,19 +157,28 @@ public final void node() throws Exception {
                 // OK
             }
 
-            try (Node node = cloudProvider.createNode("mynode")) {
-                assertEquals(1, cloudProvider.getNodes().size());
-                assertNotNull(cloudProvider.getNode("mynode"));
+            try (CreatedNodes createdNodes = cloudProvider.createNodes("mynode", "othernode")) {
```

Comment:
IMO, this parallel 2 nodes start gives nearly the same coverage as when started one by one, but with great performance gain. The logic behind `createNodes` is calling `createNode` so we only lost the very small part of assertions.

If we use only one node test, then the coverage is significantly lower.